packages/react/src/VideoPlayer/components/controls/Controls.tsx

```diff
@@ -0,0 +1,304 @@
+              fill="currentColor"
+            />
+          </svg>
+        ) : (
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="fff">
+            <path
+              fillRule="evenodd"
+              clipRule="evenodd"
+              d="M11.553 3.064C11.686 3.12273 11.799 3.21883 11.8784 3.34062C11.9577 3.46241 12 3.60464 12 3.75V20.25C12.0001 20.3954 11.9579 20.5377 11.8786 20.6596C11.7993 20.7814 11.6863 20.8776 11.5533 20.9364C11.4204 20.9952 11.2732 21.0141 11.1296 20.9908C10.9861 20.9674 10.8525 20.9029 10.745 20.805L5.46 16H2.75C2.28587 16 1.84075 15.8156 1.51256 15.4874C1.18437 15.1592 1 14.7141 1 14.25V9.75C1 8.784 1.784 8 2.75 8H5.46L10.745 3.195C10.8526 3.0974 10.9863 3.03316 11.1297 3.01008C11.2731 2.987 11.4202 3.00608 11.553 3.065V3.064Z"
+              fill="currentColor"
+            />
+            <line
+              x1="16"
+              y1="14.5962"
+              x2="20.5962"
+              y2="10"
+              stroke="currentColor"
+              strokeWidth="1.5"
+              strokeLinecap="round"
+            />
+            <line
+              x1="16.0607"
+              y1="10.0604"
+              x2="20.6569"
+              y2="14.6566"
+              stroke="currentColor"
+              strokeWidth="1.5"
+              strokeLinecap="round"
+            />
+          </svg>
+        )}
+      </IconControl>
+    ),
+    [volume, previousVolume],
+  )
+
+  /* 5. Volume                                                  */
+  useEffect(() => {
+    if (videoRef.current) videoRef.current.volume = volume
+  }, [videoRef, volume])
+
+  const renderVolume = useMemo(
+    () => (
+      <Range
+        type="range"
+        min="0"
+        max={1}
+        step={0.001}
+        onInput={e => {
+          setVolume(e.currentTarget.valueAsNumber)
+        }}
+        className={styles.VideoPlayer__volumeBar}
+        value={volume}
+        a11yStep={0.1}
+        id="VideoPlayer__Volume"
+      />
+    ),
+    [volume],
+  )
+
+  const renderFullscreenButton = useMemo(() => {
+    const handleFullScreen = () => {
+      if (videoWrapperRef.current) {
+        if (!fullScreen) {
+          videoWrapperRef.current.requestFullscreen()
+          setFullScreen(true)
+        } else {
+          document.exitFullscreen()
+          setFullScreen(false)
+        }
+      }
+    }
+
+    return (
+      <IconControl onClick={handleFullScreen} tooltip={!fullScreen ? 'Full Screen' : 'Exit Full Screen'}>
```

Comment:
```suggestion
      <IconControl onClick={handleFullScreen} tooltip={fullScreen ? 'Exit full screen' : 'Full screen'}>
```

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5